Rakefile

```
@@ -15,7 +15,8 @@ task :generate_policy_list do
   # Preparation for getting information from Github repository
   repo_name = "flexera-public/policy_templates"
   branch = "master"
-  github_client = Octokit::Client.new
+  github_api_token = `echo $GITHUB_API_TOKEN`.strip
```

```suggestion
  github_api_token = ENV["GITHUB_API_TOKEN"].strip
```
Shouldn't even need the strip imo.. but I think this will be a little safer and the value potentially less likely to get exposed in logs